python/pyarrow/__init__.py

```diff
@@ -173,6 +173,7 @@ def print_entry(label, value):
                          dictionary,
                          run_end_encoded,
                          fixed_shape_tensor,
+                         bool8,
```

Comment:
We can keep the PyArrow changes for another PR to make reviewing simpler, IMHO.

Reply:
Do you think it's ok to leave this PR as it is?